core/lib/config/src/configs/general.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{
+    configs::{
+        chain::{CircuitBreakerConfig, MempoolConfig, OperationsManagerConfig, StateKeeperConfig},
+        fri_prover_group::FriProverGroupConfig,
+        house_keeper::HouseKeeperConfig,
+        FriProofCompressorConfig, FriProverConfig, FriProverGatewayConfig,
+        FriWitnessGeneratorConfig, FriWitnessVectorGeneratorConfig, ObservabilityConfig,
+        PrometheusConfig, ProofDataHandlerConfig,
+    },
+    ApiConfig, ContractVerifierConfig, DBConfig, ETHConfig, PostgresConfig, SnapshotsCreatorConfig,
+};
+
+#[derive(Debug, PartialEq)]
+pub struct GeneralConfig {
```

Comment:
I don't really like this structure, given that it makes it possible to pass the whole `GeneralConfig` as an argument while expecting some (unknown at the call site) fields to be set. I would prefer to not have such a structure (e.g. we can deserialize each config we need from the same file).